src/locales/en/common.json

```diff
@@ -232,6 +232,7 @@
   "Estimated reward": "Estimated reward",
   "Events": "Events",
   "Execution status": "Execution status",
```

Comment:
NRTTL
I am not able to change multi sig account. I dont see the send button after going back to initiator multi sig account
<img width="1437" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/4a65313f-3e0f-4783-ace8-e30f397f208e">

Reply:
This now works!